spec/rails_helper.rb

```diff
@@ -55,6 +55,7 @@
 RSpec.configure do |config|
   # add `FactoryBot` methods
   config.include FactoryBot::Syntax::Methods
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
Which test is using this?

Reply:
None yet, I tried using it in the feature specs but I was having trouble and had to stick with Timecop. I can remove it if you want